src/kafka-util/src/bin/kgen.rs

```diff
@@ -504,6 +511,16 @@ async fn main() -> anyhow::Result<()> {
                 .takes_value(true)
                 .default_value("localhost:9092"),
         )
+        .arg(
+            Arg::with_name("schema-registry")
+                .short("s")
+                .long("schema-registry")
+                .takes_value(true)
+                .default_value("http://localhost:8081"),
+                // .conflicts_with("values", "bytes"),
+                // conflicts_with can only take the argument name, not value
+                // perhaps we can do [--avro, --bytes] instead of --values and then conflicts_with("bytes")
```

Comment:
Don't be afraid to just stick an `if` statement after the option parsing! There's only so much these magic clap configurations can do IME.

Reply:
Sounds good, will do!